__config__/prometheus-config.production.json

```diff
@@ -64,6 +64,13 @@
     "help": "Number of errors per second per request",
     "type": "Meter"
   },
+  {
+    "name": "trace_duration_seconds",
+    "help": "The time in seconds for traces to complete",
+    "type": "Histogram",
+    "labelNames": ["span"],
+    "buckets": [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
+  },
```

Comment:
Is it worth adding this to the `koa-prometheus` library directly?
https://github.com/uswitch/koa-core/blob/master/packages/koa-prometheus/src/koa-prometheus.standards.json

Reply:
Yes, I think you're right - are most people using the standards JSON?

Reply:
Good point, the default is `false` for loading standards - maybe worth adding it to the default one then